internal/resources/slo/resource_slo.go

```diff
@@ -5,14 +5,21 @@ import (
 	"fmt"
 	"regexp"
 
-	gapi "github.com/grafana/grafana-api-golang-client"
+	slo "github.com/grafana/slo-openapi-client/go"
 
 	"github.com/grafana/terraform-provider-grafana/internal/common"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
 )
 
+const (
+	QueryTypeFreeform  string = "freeform"
```

Comment:
question: No way to extract this from the generated API client?

Reply:
Will get it through a follow-up